src/style.css

```diff
@@ -1 +1,122 @@
-/* empty source */
+/* #region general */
+html {
+  font-family: "Roboto", Arial, sans-serif;
+  font-weight: 400;
+  line-height: 14px;
+  font-size: 10px;
+  color: #616070;
+}
+
+body {
+  margin: 0;
+}
+/* #endregion */
+
+/* #region card */
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+/* #endregion */
+
+/* #region productInfo */
+.card__image {
+  height: 134px;
+  margin: 0 auto 40px;
+  display: block;
+  width: 160px;
+}
+
+.card__details {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: #060b35;
+}
+
+.card__code {
+  margin-bottom: 16px;
+}
+
+/* #region stars */
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.reviews {
+  color: #060b35;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 > :first-child,
+.stars--2 > :nth-child(-n+2),
+.stars--3 > :nth-child(-n+3),
+.stars--4 > :nth-child(-n+4),
+.stars--5 > .stars__star {
+  background-image: url(./images/star-active.svg);
+}
+/* #endregion */
+
+.card__price-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__price {
+  font-size: 16px;
+  font-weight: 700;
+  color: #060b35;
+  line-height: 18px;
+}
+
+.card__buy {
+  height: 40px;
+  border-radius: 5px;
+  background-color: #00acdc;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-decoration: none;
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+  text-transform: uppercase;
+  line-height: 16px;
+  box-sizing: border-box;
```

Comment:
please group box model rules at the top

Reply:
Thanks @mgrinko . I have noted all your comments and will try to use them in the next tasks.